### ISSUE TITLE
Wii U freezes when starting Loadiine

### ISSUE THREAD

**Issue by hex7cd** [dimok789/loadiine_gx2#166]
My Wii U runs ColdBoot HaxChi.
Tried running it from web exploit as well as the Homebrew Launcher channel.

Immediately after the .elf is loaded my Wii U freezes with a black screen.

**Comment by NinPX** [comment]
Me too I need help

**Comment by JayCanuck** [comment]
Same here

**Comment by hex7cd** [comment]
I don't think that anything's going to happen though. Way too long since this was last worked on.

**Comment by JayCanuck** [comment]
Just saw [this post](https://gbatemp.net/threads/loadiine-gx2.413823/page-293#post-7218665) which mentioned deleting the `./wiiu/controller` directory to fix the issue. Happy to report that sucessfully fixed the issue for me. Hopefully will work for you guys too.

**Comment by Maschell** [comment]
I can confirm that the controller folder is an issue if you have too many file in it

**Comment by guyluke** [comment]
Hey guys,
I've been following this issue for a while since I was getting a lot of problems running games on loadiine. 
But when I finally discovered  the so called Brazilian Method that will use Wii U USB Helper that will download and patch titles directly from Nintendo and WUP Installer that will install then on your Console. Is so much easier and more reliable I never had the need to use loadiine again.
I just wanted to let you know this exists, because at the beginning I ran in to a lot of trouble trying to find updated information about the WiiU Homebrew. 
Anyway, sorry if I'm off topic. Thanks guys! :)

**Comment by videofindersTV** [comment]
I also had the same problem, but now I fixed it.
Removing the `sd:/wiiu/controller/` folder makes Loadiine load.
Thank you, @JayCanuck !

**Comment by bsfez** [comment]
THANK YOU !!! @JayCanuck !

**Comment by WiisleyU** [comment]
My Loadiine starts fine but ALWAYS will crash opening ANY game, I tried it to play DS and N64 games but it not worked at all.

**Comment by KingGhidorah97** [comment]
Made an account just to say thanks @JayCanuck :)

**Comment by Maschell** [comment]
Commit 33ea151 should (finally) fix the problem. Check out the newest nightly

**Comment by Chentufio** [comment]
I fixed the problem of the controller folder moving that folder to the sd root folder and then create a direct access in the wiiu folder so you can use the hid to vpad program without actually deleting the controller folder.

**Comment by Alan-Cha** [comment]
@Chentufio Could you provide more detail about what you did? I'm not sure what you mean by direct access...

**Comment by Chentufio** [comment]
> 
> 
> @Chentufio Could you provide more detail about what you did? I'm not sure what you mean by direct access...

Sorry, by "direct access" I meant to say that I created a shortcut as in Windows when you create an access to a folder in other direction. That worked for me at the time.

**Comment by Alan-Cha** [comment]
@Chentufio Thank you so much for replying! I didn't see a lot of activity on your account so I didn't have high hopes. I really appreciate it! 

To be honest, I'm not too familiar with Windows functionality. Are you talking about something like [this](https://stackoverflow.com/questions/1169556/making-a-windows-shortcut-start-relative-to-where-the-folder-is)?

Again, thanks for the response!

**Comment by Alan-Cha** [comment]
I can confirm that creating a "shortcut" works. I can run Loadiine and HIDtoVPAD (separately) by moving the `sd:/wiiu/controller` folder to `sd:/controller` and creating a shortcut from `sd:/wiiu` to `sd:/controller`.

**Comment by Chentufio** [comment]
> 
> 
> @Chentufio Thank you so much for replying! I didn't see a lot of activity on your account so I didn't have high hopes. I really appreciate it!
> 
> To be honest, I'm not too familiar with Windows functionality. Are you talking about something like [this](https://stackoverflow.com/questions/1169556/making-a-windows-shortcut-start-relative-to-where-the-folder-is)?
> 
> Again, thanks for the response!

Yeah, good to hear it worked. Sorry for answering just now, but the thread you asked it´s exactly what to do to the folder.

Cheers.
